main.py

```diff
@@ -8,11 +8,12 @@
 
 def get_season(now):
   Y = 2000  # dummy leap year to allow input XXXX-02-29 (leap day)
-  seasons = [('❄️winter', (date(Y, 1, 1), date(Y, 3, 20))),
-             ('🌷spring', (date(Y, 3, 21), date(Y, 6, 20))),
-             ('🌞summer', (date(Y, 6, 21), date(Y, 9, 22))),
-             ('🍂fall', (date(Y, 9, 23), date(Y, 12, 20))),
-             ('❄️winter', (date(Y, 12, 21), date(Y, 12, 31)))]
+  # in this house we subscribe to the quarter system 
```

Comment:
yeah so we're gonna need per-user season systems, ideally with an option for users to set their preferred system